### ISSUE TITLE
Improve ensemble strategy win rate and signal quality

### ISSUE THREAD

**Issue by Pr0mp7** [Photonensauger/Quant-2.0#6]
## Problem

The ensemble strategy on GLD (10yr training, 2yr backtest) produces a **48.33% win rate** with a profit factor of only 1.06. The strategy is barely better than a coin flip, making it unprofitable after transaction costs.

### Current Backtest Results (GLD)

| Metric | Value |
|--------|-------|
| Total Return | -0.37% |
| Sharpe Ratio | -0.02 |
| Win Rate | 48.33% |
| Profit Factor | 1.06 |
| Trades | 180 |
| Max Drawdown | 3.88% |

### Root Causes

1. **Momentum model is a coin flip** — 51.58% directional accuracy during training, dragging down the ensemble (other models: 58-63%)
2. **Equal weighting** — all 4 models start with equal weight (0.25). A bad model has the same influence as a good one until weight updates kick in
3. **Short forecast horizon** — `forecast_horizon=5` (5 bars = 1 trading week) is too short for GLD, which trends over weeks/months
4. **Single-asset training** — models only see GLD patterns, missing cross-asset context (USD, real yields, SPY risk-off flows)
5. **High minimum confidence** — `min_confidence=0.6` filters out moderate-conviction signals that could still be profitable
6. **Conservative signal gates** — `confirmation_bars=2` + `signal_cooldown=5` means the strategy waits 2 bars to confirm direction then sits out 5 bars. On a trending asset this misses entries

---

### 1. Drop Momentum from ensemble (CLI — no code change)

Use `--models transformer,itransformer,lstm` to exclude the worst-performing model.

**Expected impact:** Removes the 51% accuracy model from the vote, improving consensus quality.

### 2. Initialize ensemble weights by validation accuracy (code change)

Instead of equal `1/n` weights, seed initial weights proportional to each model's directional accuracy from training.

**File:** `quant/strategies/ensemble.py`

Example: Transformer 62%, iTransformer 59%, LSTM 63% → weights: 0.34, 0.32, 0.34 instead of 0.33, 0.33, 0.33.

### 3. Increase forecast horizon to 10 bars (config change)

Change `forecast_horizon` from 5 to 10 in both `FeatureConfig` and `ModelConfig`. Gold trends over 2-week cycles, not 1-week.

**File:** `quant/config/settings.py` (two places)

**Requires retraining** all models after the change.

### 4. Lower learning rate to 5e-4 (CLI — no code change)

Use `--lr 0.0005` during training. The current 1e-3 may cause models to overshoot optima on smooth price data like GLD.

### 5. Tune signal filter parameters (config change)

Relax the trading gates to let more signals through:

| Parameter | Current | Proposed | Rationale |
|-----------|---------|----------|-----------|
| `min_confidence` | 0.6 | 0.4 | Let moderate-conviction trades through |
| `signal_cooldown` | 5 | 3 | Less wait between trades |
| `confirmation_bars` | 2 | 1 | Don't wait for confirmation on trending assets |

**File:** `quant/config/settings.py` → `TradingConfig`

### 6. Multi-asset training (CLI — no code change)

Train on correlated assets together:
```bash
python scripts/train_model.py --model all --assets GLD,SPY,TLT --interval 1d --epochs 50 --days 3650
```

The feature pipeline already generates per-asset features. Training on GLD+SPY+TLT gives models context about risk-on/risk-off regimes.

---

## Implementation Order

1. Config changes (forecast horizon, signal filters) — `settings.py`
2. Ensemble weight initialization — `ensemble.py`
3. Retrain all models with `--lr 0.0005 --epochs 50 --days 3650 --models transformer,itransformer,lstm`
4. Backtest and compare

## Success Criteria

- Win rate > 52%
- Profit factor > 1.2
- Sharpe > 0.3
- Fewer than 10 NaN/Inf loss events during training
